app/screens/Lists/BooksList/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { View, FlatList } from 'react-native';
+
+import BookItem from './components/BookItem';
+import { bookData } from './constants';
+
+class BooksListContainer extends Component {
+  renderList = item => (
+    <BookItem
+      image={item.image_url || undefined}
```

Comment:
undefined or null its the same to get the default props of the component

Reply:
The app brakes when i return a null

Reply:
yes, because you need set a default props on the image